src/platform/web/ui/css/themes/element/theme.css

```diff
@@ -18,10 +18,6 @@ limitations under the License.
 @import url('inter.css');
 @import url('timeline.css');
 
-:root {
-    font-size: 10px;
```

Comment:
As noted on the ticket, we can't remove this until Hydrogen doesn't use `rem` anymore as a unit in CSS. The PR will currently look like this, with fonts too large:

![image](https://user-images.githubusercontent.com/274386/135408521-8be79865-5a30-4eec-a491-52396f39ab6e.png)